pt/lessons/specifics/mnesia.md

```diff
@@ -0,0 +1,185 @@
+  --sname "name"    Makes and assigns a short name to the distributed node
+  --cookie "cookie" Sets a cookie for this distributed node
+  --hidden          Makes a hidden node
+  --werl            Uses Erlang's Windows shell GUI (Windows only)
+  --detached        Starts the Erlang VM detached from console
+  --remsh "name"    Connects to a node using a remote shell
+  --dot-iex "path"  Overrides default .iex.exs file and uses path instead;
+                    path can be empty, then no file will be loaded
+
+** Options marked with (*) can be given more than once
+** Options given after the .exs file or -- are passed down to the executed code
+** Options can be passed to the VM using ELIXIR_ERL_OPTIONS or --erl
+```
+
+Quando passamos a opção `--help` para IEx a partir da linha de comando, nos é apresentado todas as opções possíveis. Podemos ver que existe opção `--name` e `--sname` para atribuição de informações para nós. Um nó é apenas uma Máquina Virtual do Erlang lidando com suas próprias comunicações, bargabe collection, processamento agendado, memória e muito mais. O nó está sendo nomeado como **nonode@nohost** simplesmente por padrão.
+
+```shell
+$ iex --name learner@elixirschool.com
+
+Erlang/OTP 18 [erts-7.2.1] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
+
+Interactive Elixir (1.2.1) - press Ctrl+C to exit (type h() ENTER for help)
+iex(learner@elixirschool.com)> Node.self
+:"learner@elixirschool.com"
+```
+
+Como podemos ver agora, o nó que estamos executando é um átomo chamado `:"learner@elixirschool.com"`. Se nós executarmos `Mnesia.create_schema([node()])` novamente, nós iremos ver que ele criou uma outra pasta chamada **Mnesia.learner@elixirschool.com**. O propósito disto é bem simples. Nós em Erlang são usados para conectar a outros nós para compartilhar (distribuir) informação e recursos. Isto não tem que estar limitado a mesma máquina e pode comunicar através de LAN, internet, etc.
+
+## Iniciando Mnesia
+
+Agora que temos o conhecimento básico e criação do banco de dados, estamos agora em posição de iniciar o Mnesia DMBs com o comando ```Mnesia.start/0```.
+
+```shell
+iex> alias :mnesia, as: Mnesia
+iex> Mnesia.create_schema([node])
+:ok
+iex> Mnesia.start()
+:ok
+```
+
+Vale a pena manter em mente que quando executando um sistema distribuído com dois ou mais nós participando, a função `Mnesia.start/1` deve ser executada em todos os nós participantes.
+
+## Criando Tabelas
+
+A função `Mnesia.create_table/2` é usada para criar tabelas dentro do nosso banco de dados. Abaixo criamos uma tabela chamada `Person`, e em seguida, passamos uma lista de palavra-chave definindo o schema da tabela.
+
+```shell
+iex> Mnesia.create_table(Person, [attributes: [:id, :name, :job]])
+{:atomic, :ok}
+```
+Nós definimos as colunas usando os átomos `:id`, `:name`, e ':job'. Quando executamos `Mnesia.create_table/2`, ele irá retornar qualquer uma das seguinte respostas:
```

Comment:
`:job` atom is with single quotes instead of backticks. Typo from the original text?